src/components/Box/Box.stories.mdx

```diff
@@ -864,7 +867,10 @@ The box will have different margin and padding values for at the base, tablet, a
 #### Direction
 
 <Canvas>
-  <Story name="responsive direction">
+  <Story
+    name="responsive direction"
+    parameters={{ chromatic: { viewports: [320, 700, 1012, 1300] } }}
```

Comment:
@juanfabrega not sure if there's a way to use the `BREAKPOINT_VALUES` defined in tokens.js and add 20, so the viewports array is just hardcoded.

Reply:
@nathanyoung I do believe there is a better way to do this, and make it available globally as a storybook decorator/setting. It would take maybe 1-2 hours between implementation and testing, I think. For now if you want to merge as is I don't think that's an issue.

Reply:
going to do the above as a separate change.